src/pages/ReimbursementAccount/BankInfo/substeps/Confirmation.tsx

```diff
@@ -70,48 +58,34 @@ function Confirmation({reimbursementAccount, reimbursementAccountDraft, onNext}:
-                <View style={[styles.cardSection]}>
-                    {setupType === CONST.BANK_ACCOUNT.SUBSTEP.MANUAL && (
-                        <View style={[styles.mb5, styles.flexRow, styles.alignItemsCenter]}>
-                            <Icon
-                                src={Expensicons.Bank}
-                                additionalStyles={[styles.confirmBankInfoCompanyIcon, styles.mr3]}
-                                fill={theme.iconHovered}
-                            />
-                            <View>
-                                <View style={[styles.mb3]}>
-                                    <Text style={[styles.mutedTextLabel, styles.mb1]}>{translate('bankAccount.routingNumber')}</Text>
-                                    <Text style={styles.confirmBankInfoNumber}>{values[bankInfoStepKeys.ROUTING_NUMBER]}</Text>
-                                </View>
-                                <View>
-                                    <Text style={[styles.mutedTextLabel, styles.mb1]}>{translate('bankAccount.accountNumber')}</Text>
-                                    <Text style={styles.confirmBankInfoNumber}>{values[bankInfoStepKeys.ACCOUNT_NUMBER]}</Text>
-                                </View>
-                            </View>
-                        </View>
-                    )}
-                    {setupType === CONST.BANK_ACCOUNT.SUBSTEP.PLAID && (
-                        <MenuItem
-                            interactive={false}
-                            icon={Expensicons.Bank}
-                            iconType={CONST.ICON_TYPE_ICON}
-                            iconStyles={[styles.confirmBankInfoCompanyIcon]}
-                            iconFill={theme.iconHovered}
-                            wrapperStyle={[styles.pl0, styles.mb6]}
-                            title={values[bankInfoStepKeys.BANK_NAME]}
-                            description={`${translate('bankAccount.accountEnding')} ${(values[bankInfoStepKeys.ACCOUNT_NUMBER] ?? '').slice(-4)}`}
+                {setupType === CONST.BANK_ACCOUNT.SUBSTEP.MANUAL && (
+                    <View style={[styles.mb5]}>
+                        <MenuItemWithTopDescription
+                            description={translate('bankAccount.routingNumber')}
+                            title={values[bankInfoStepKeys.ROUTING_NUMBER]}
+                            shouldShowRightIcon={!Boolean(bankAccountID)}
+                            onPress={handleModifyAccountNumbers}
                         />
-                    )}
-                    <Text style={[styles.confirmBankInfoText, styles.mb4]}>{translate('bankAccount.thisBankAccount')}</Text>
-                    <MenuItem
-                        icon={Expensicons.Bank}
-                        iconType={CONST.ICON_TYPE_ICON}
-                        title={translate('bankAccount.connectDifferentAccount')}
```

Comment:
What is the reason of removing that, there are business requirements changes?

Reply:
https://github.com/barttom/App-expensify/pull/5#issuecomment-1900511114